assets/prefabs/Blackberry/Blackberry.prefab

```diff
@@ -0,0 +1,10 @@
+{
+  "parent": "engine:iconItem",
+  "DisplayName": {
+    "name": "Blackberry"
+  },
+  "Item": {
+    "icon": "SimpleFarming:SimpleFarming#Blackberry",
+    "stackId": "Blackberry"
+  }
+}
```

Comment:
You could add newlines to each text file like this where you don't yet have a blank line at the bottom. Not critical but a good practice.